config/routes.rb

```diff
@@ -53,6 +53,8 @@
   match 'home' => 'home#front'
   resources :relationships, only: [:create, :destroy]
 
+  get '/wiki/:id/comments', to: 'wiki#wiki_comments'
```

Comment:
Oh my bad, this `wiki_comment` were used when the action was within `comments_controller`. @jywarren is right. As this is within `wiki_controller` just `comments` is enough and meaningfull. 
Thanks.